### PR TITLE
Introduce K5 capability for E2E Student

### PR DESCRIPTION
**Pull request by davidvarga93**
refs: MBL-15378
affects: Student
release note: none

test plan:

### CODE REVIEW

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Build failed, skipping coverage check
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


[MBL-15378](https://instructure.atlassian.net/browse/MBL-15378)

#### Affected Apps: Student
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Canvas.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 1]
```diff
@@ -9,15 +9,6 @@
...
         }
-      },
-      {
-        "package": "Swifter",
-        "repositoryURL": "https://github.com/httpswift/swifter",
-        "state": {
-          "branch": "stable",
-          "revision": "e9b1d5ef5a8683babe4265007779a87c324abf8a",
-          "version": null
-        }
```
Comment (vargaat) [inline Canvas.xcworkspace/xcshareddata/swiftpm/Package.resolved:20]: The project needs this package.

**Review by vargaat — CHANGES_REQUESTED** [review]

**Review by vargaat — CHANGES_REQUESTED** [review]
Seems like the TestFoundation project file is damaged.

`<IDEFileReference, 0x7ff6de743020: name:TestsFoundation path:group:TestsFoundation/TestsFoundation.xcodeproj> Failed to load container at path: /Users/vagrant/git/TestsFoundation/TestsFoundation.xcodeproj, Error: Error Domain=Xcode3ProjectErrorDomain Code=2 "The project ‘TestsFoundation’ is damaged and cannot be opened due to a parse error. Examine the project file for invalid edits or unresolved source control conflicts.
`

**Comment by inst-danger** [comment]
<details><summary>Student.ipa</summary>
  <a href="https://app.bitrise.io/artifact/86660872/p/b6a23a514bc0529b96d1893bfb02f3ab">
    
  </a>
</details>
<details><summary>Teacher.ipa</summary>
  <a href="https://app.bitrise.io/artifact/86660877/p/f5165278ee64afc8aed224622f43b5d0">
    
  </a>
</details>
<details><summary>Parent.ipa</summary>
  <a href="https://app.bitrise.io/artifact/86660869/p/8864012f82d52a2061365d1cf1f93a27">
    
  </a>
</details>

**Comment by davidvarga93** [comment]
Nightly:
https://app.bitrise.io/build/651cff96-373a-4cfe-95b5-0ede997e148f#?tab=log

**Review by vargaat — APPROVED** [review]

**Review by idevainstr — APPROVED** [review]

**Comment by davidvarga93** [comment]
K5StudentE2ETests
    ✔ testK5Reset (16.897 seconds)
    ✔ testStudentK5 (16.749 seconds)

**Pull request merged**